### DISCUSSION TITLE
Docker I use  -u "$(id -u):$(id -g)"   -e "DOCKER_USER=$USER" and can't stop start it

### DISCUSSION THREAD

**Discussion by roroyu** [coder/code-server#2244] [Q&A]
Docker 
I use 
 -u "$(id -u):$(id -g)" \
  -e "DOCKER_USER=$USER"
and when I stop it, docker start it
is exit by usermod: user 'coder' does not exist

**Comment by roroyu** [comment]
maybe I use this before restart

sudo groupadd docker -g 999
sudo usermod -aG docker roro

**Comment by roroyu** [comment]
I try it again
Maybe just use -e "DOCKER_USER=$USER"
And cannot restart docker

**Comment by code-asher** [comment]
What's the full command you're running?
